aten/src/ATen/cuda/CUDASparseBlas.h

```diff
@@ -315,6 +315,33 @@ template <>
 void bsrsm2_solve<c10::complex<double>>(
     CUSPARSE_BSRSM2_SOLVE_ARGTYPES(c10::complex<double>));
 
+#define CUSOLVER_LINEAR_SOLVE_ARGTYPES(scalar_t)                                 \
```

Comment:
This file should be used only for the cuSPARSE library. There's CUDASolver.h for cuSOLVER library, so could you please move definitions and implementations there?

Reply:
Yes, done! Thank you for the pointer.